op-acceptance-tests/tests/interop/message/interop_msg_test.go

```diff
@@ -588,6 +588,9 @@ func TestExecMessageInvalidAttributes(gt *testing.T) {
 		{mismatchedLogIndex}, {mismatchedTimestamp}, {msgNotPresent}, {logIndexGreaterOrEqualToEventCnt},
 	}
 
+	// save correct nonce to avoid flakiness while validating message
+	nonce := bob.PendingNonce()
```

Comment:
There is a time gap between invalid validating message is dropped from the op-geth txpool, while not dropped yet, the client calls pendingNonceAt which is a nonce which eventually becomes non-consecutive nonce after previous invalid tx has dropped, causing the valid tx not to be included in the block.